package.json

```diff
@@ -17,7 +17,7 @@
   ],
   "scripts": {
     "format": "prettier --loglevel warn --write \"**/*.{js,ts,tsx,css,md}\"",
-    "test": "echo \"Error: no test specified\" && exit 1",
+    "test": "echo \"No test specified\"",
     "build": "rimraf ./dist && tsc",
     "preversion": "npm run build && git commit --no-verify --allow-empty -am \"Update dist\"",
```

Comment:
Let's drop `preversion` as it's redundant now as far as I understand.

Reply:
https://github.com/pmedianetwork/webpack-config/pull/9/commits/6508f73afc06e651e83e5a3d2b973c147b199cd6